introduction.md

```diff
@@ -0,0 +1 @@
+My name is DSuarez, and I am not an alcoholic. I expect all of you to say hi to me in a dreary tone.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.